front/app/containers/Admin/projects/project/analysis/Insights/SummarizeButton.tsx

```diff
@@ -61,13 +65,15 @@ const SummarizeButton = () => {
         processing={isLoadingPreCheck || isLoadingSummary}
         whiteSpace="wrap"
       >
-        Auto-summarize
+        {formatMessage(translations.summarize)}
         <br />
         <Text fontSize="s" m="0" color="grey600">
           {summaryPossible && summaryAccuracy && (
-            <>{summaryAccuracy * 100}% accuracy</>
+            <>
+              {summaryAccuracy * 100}% {formatMessage(translations.accuracy)}
```

Comment:
There could be languages where the order of this is inversed, or % formatting is done differently. Maybe rather do the percentage with an interpolation?